spec.md

```diff
@@ -26,6 +26,11 @@ Enter CloudEvents, a specification for describing event data in a common way.
 CloudEvents seeks to ease event declaration and delivery across services,
 platforms and beyond.
 
+The [Serialization Profile](serialization.md) specified how to
```

Comment:
This might be better as "specifies". (Or am I being too tense?)

Reply:
No, you're right - just a disconnect between the brain and fingers.